contrib_bots/lib/github.py

```diff
@@ -0,0 +1,22 @@
+# The purpose of this file is to handle all requests
+# to the Github API, which will make sure that all
+# requests are to the same account, and that all requests
+# authenticated correctly and securely
+# The sole purpose of this is to authenticate, and it will
+# return the requests session that is properly authenticated
+import requests
+import six.moves.configparser, os
+
+def auth():
+    # Sets up the configParser to read the .conf file
+    config = six.moves.configparser.ConfigParser()
+    # Reads the config file
+    config.read([os.path.expanduser('~/.zuliprc-local')])
```

Comment:
I know there's been some confusion around where we put the config info for github stuff, but I believe that putting it into `~/.zuliprc-local` is **not** the way to go.  One pitfall here is that a user might change their API key in Zulip and download a new zuliprc file, which will overwrite the github keys.